scripts/firedrake-install

```diff
@@ -130,6 +133,7 @@ else:
     args.adjoint = args.adjoint or args.install_adjoint
     args.slepc = False
     args.slepc = args.slepc or args.install_slepc
+    args.virtualenv_name = False
```

Comment:
So first we run the old firedrake-update, which does not have `virtualenv_name` in its `args`, so the initial update script rebuild won't replace `False` with `"firedrake"`. When the updated script runs, it fails because `False/bin/`etc. doesn't work.